$OpenBSD$

It should be possible to get linux-v4l2 working with a bit more effort.
ffmpeg can capture from these sources anyway (source format video4linux2).

--- plugins/CMakeLists.txt.orig	Sat Dec 24 12:44:14 2016
+++ plugins/CMakeLists.txt	Mon Jan  9 17:52:38 2017
@@ -44,6 +44,10 @@ elseif("${CMAKE_SYSTEM_NAME}" MATCHES "FreeBSD")
 	add_subdirectory(linux-pulseaudio)
 	add_subdirectory(linux-v4l2)
 	add_subdirectory(linux-jack)
+elseif("${CMAKE_SYSTEM_NAME}" MATCHES "OpenBSD")
+	add_subdirectory(linux-capture)
+	add_subdirectory(linux-pulseaudio)
+	add_subdirectory(linux-jack)
 endif()
 
 if(WIN32 OR APPLE)
